res/app/control-panes/dashboard/enrollment/enrollment-spec.js

```diff
@@ -0,0 +1,20 @@
+describe('ShellCtrl', function() {
```

Comment:
one more thing, I'm not sure about this `ShellCtrl` whether it would not somehow collide with the Shell widget. TBH I don't understand what is this for, so maybe it doesn't matter, but can you pls check? What happens if you rename it e.g. to `EnrollmentCtrl`, will it still work? If yes, I would rather change it to prevent some interference between widgets